product_docs/docs/biganimal/release/overview/replication.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Faraway replicas"
+---
+
+You can create (faraway read) replicas of your *single node* and *high availability* clusters. In case of a failure, you can manually promote the faraway replica to a full-fledged BigAnimal cluster making it capable of accepting writes. 
```

Comment:
```suggestion
You can create faraway read replicas (faraway replicas) of your *single node* and *high availability* clusters. In case of a failure, you can manually promote the faraway replica to a full-fledged BigAnimal cluster making it capable of accepting writes. 
```